pkg/orchestration/wiring/wiringutil/knownshapes/rds.go

```diff
@@ -0,0 +1,42 @@
+package knownshapes
+
+import (
+	"github.com/atlassian/voyager/pkg/orchestration/wiring/wiringplugin"
+)
+
+const (
+	RDSShape wiringplugin.ShapeName = "voyager.atl-paas.net/RDS"
+)
+
+// +k8s:deepcopy-gen=true
+// +k8s:deepcopy-gen:interfaces=github.com/atlassian/voyager/pkg/orchestration/wiring/wiringplugin.Shape
+type RDS struct {
+	wiringplugin.ShapeMeta `json:",inline"`
+}
+
+// NewRDS creates a new RDS
+func NewRDS() *RDS {
+	return &RDS{
+		ShapeMeta: wiringplugin.ShapeMeta{
+			ShapeName: RDSShape,
+		},
+	}
+}
+
+// Name returns the RDS ShapeName
+func (a *RDS) Name() wiringplugin.ShapeName {
+	return a.ShapeName
+}
+
+// FindRDSShapes returns a single instance of the RDS shape if found and will error if there are multiples
+func FindRDSShapes(shapes []wiringplugin.Shape) (*RDS, bool /*found*/, error) {
```

Comment:
This is a copy-pasta from the ASAP shape: https://github.com/atlassian/voyager/blob/master/pkg/orchestration/wiring/wiringutil/knownshapes/asap_key.go#L32-L42